File: MatricesJava/src/Ejercicio1.java

package paquete;

import java.util.Scanner;

public class Ejercicio1 {
    public static void main(String[] args){
        /*Hacer un programa que almacene números en una matriz de 5*6. Imprimir la
        suma de los numeros almacenados en la matriz.
        */
        Scanner scanner = new Scanner(System.in);
        float sumaNumerosMatriz = 0;
        System.out.print("Ingrese el numero de filas: "); int cantidadFilas = scanner.nextInt();
        System.out.print("Ingrese el numero de columnas: "); int cantidadColumnas = scanner.nextInt();
        float matriz[][] = new float[cantidadFilas][cantidadColumnas];
        for (int i = 0; i < matriz.length; i++) {
            for (int j = 0; j < matriz[i].length; j++) {
                System.out.print("Ingrese un numero para la posicion: (" + (i+1)
                + "-" + (j+1) + "): "); sumaNumerosMatriz += matriz[i][j] = scanner.nextFloat();
            }
        }
        System.out.println("La suma de los numeros de la matriz es: " + sumaNumerosMatriz);
    }
}

File: MatricesC++/Ejercicio1.cpp
#include<iostream>
#include<conio.h>

using namespace std;

int main(){
	/*Hacer un programa que almacene números en una matriz de 5*6. Imprimir la suma de los números almacenados en la matriz*/
	int matriz[10][10];
	int cantidadFilas  = 0, cantidadColumnas = 0, sumaElementosMatriz = 0; 
	cout<<"Por favor ingrese la cantiadd de filas: "; cin >> cantidadFilas; 
	cout<<"Por favor ingrese la cantidad de columnas: "; cin >> cantidadColumnas;
	for(int i = 0; i < cantidadFilas; i++){
		for(int j = 0; j < cantidadColumnas; j++){
			cout<<"Por favor ingrese un numero para la posicion: ("<<i+1<<"-"<<j+1<<"): "; cin >> matriz[i][j];
			sumaElementosMatriz += matriz[i][j];
		}
	}	
	cout<<"La suma de los elementos de la  matriz es: "<< sumaElementosMatriz;
	getch();
	return 0; 
}
